pyuvdata/parameter.py

```diff
@@ -734,9 +734,12 @@ def __init__(
         value=None,
         spoof_val=None,
         description="",
+        frame="itrs",
         acceptable_range=(6.35e6, 6.39e6),
         tols=1e-3,
     ):
+        if frame == "mcmf":
+            acceptable_range = (1717100.0, 1757100.0)
```

Comment:
I'd prefer to set the default to `None` and then if it is `None`, set it to the right value based on the frame (the existing default for `itrs` and this for `mcmd`. Otherwise the user cannot override it.

Reply:
done as suggested!